openvdb_houdini/houdini/SOP_OpenVDB_Advect_Points.cc

```diff
@@ -1102,18 +1107,26 @@ VDB_NODE_OR_CACHE(VDB_COMPILABLE_SOP, SOP_OpenVDB_Advect_Points)::evalAdvectionP
     UT_String ptGroupStr;
     evalString(ptGroupStr, "group", 0, now);
 
-    parms.mPointGroup = parsePointGroups(ptGroupStr, GroupCreator(gdp));
-    parms.mGroup = matchGroup(*parms.mPointGeo, evalStdString("group", now));
+    if (advectVdbPoints)
+    {
+	parms.mPointGroup = 0;
+	parms.mGroup = matchGroup(*parms.mPointGeo, ptGroupStr);
 
-    const std::string groups = evalStdString("vdbpointsgroups", now);
+	if (!parms.mPointGroup && ptGroupStr.length() > 0) {
```

Comment:
mPointGroup is explicitly set to 0 above, so the first part of this conditional will always evaluate to true (!parms.mPointGroup)